### PR TITLE
Add optlang

### PR DESCRIPTION
**Pull request by druvus**

### CODE REVIEW

**Comment by mbargull** [comment]
Hi @druvus, if I understood correctly, you can use `noarch: python` without a `skip: True  # [win]` if you use `[skip appveyor]` in your commit message.

**Comment by druvus** [comment]
thanks @mbargull ! I didn't think about this way to do it.

File: recipes/optlang/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
```
Comment (mbargull) [inline recipes/optlang/meta.yaml]: Acc. to `setup.py` only
```yaml
build:
  - python
  - setuptools
```
should be required.

File: recipes/optlang/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+test:
+  imports:
+    - optlang
+    - optlang.tests
+    - optlang.tests.data
+
+about:
+  home: https://github.com/biosustain/optlang
+  license: Apache
+  license_family: APACHE
```
Comment (dougalsutherland) [inline recipes/optlang/meta.yaml]: Unimportant nit: `Apache`.

File: recipes/optlang/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  license_file: LICENSE
+  summary: 'optlang - sympy based mathematical programming language'
+
```
Comment (dougalsutherland) [inline recipes/optlang/meta.yaml]: Unimportant nit: you could drop this blank line here.

File: recipes/optlang/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+test:
+  imports:
+    - optlang
+    - optlang.tests
+    - optlang.tests.data
+
+about:
+  home: https://github.com/biosustain/optlang
+  license: Apache
```
Comment (dougalsutherland) [inline recipes/optlang/meta.yaml]: `Apache-2.0` please.

**Comment by druvus** [comment]
@dougalsutherland all your comments are now fixed so this should be ready.

**Comment by dougalsutherland** [comment]
Merged. Thanks @druvus!

**Pull request merged**